### ISSUE TITLE
java.lang.ClassCastException occurs when apache httpclient called

### ISSUE THREAD

**Issue by mr3** [arextest/arex-agent-java#150]
**Bug description**
java.lang.ClassCastException occurs when apache httpclient called

**Steps to reproduce**
```xml
<!-- https://mvnrepository.com/artifact/org.apache.httpcomponents/fluent-hc -->
<dependency>
    <groupId>org.apache.httpcomponents</groupId>
    <artifactId>fluent-hc</artifactId>
    <version>4.5.2</version>
</dependency>

```

Please provide [GitHub address] to reproduce this issue.

**Expected behavior**

Don't throw exceptions.

**What went wrong?**
java.lang.ClassCastException are thrown.

**Environment**

Compiler: jdk1.8.0_302
arex-agent-java: 0.2.0

**Additional context**
If there is an exception, please attach the exception trace:

Stacktrace
```
java.lang.ClassCastException: org.apache.http.client.fluent.InternalHttpRequest cannot be cast to org.apache.http.client.methods.HttpRequestBase
	at io.arex.inst.httpclient.apache.common.ApacheHttpClientAdapter.(ApacheHttpClientAdapter.java:31)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:157)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
	at org.apache.http.client.fluent.Request.internalExecute(Request.java:173)
	at org.apache.http.client.fluent.Request.execute(Request.java:177)
```

**Comment by MentosL** [comment]
I want to reproduce the problem, but I can't reproduce it. Can you provide a simple demo

```xml
    <dependencies>
        <!-- https://mvnrepository.com/artifact/org.apache.httpcomponents/fluent-hc -->
        <dependency>
            <groupId>org.apache.httpcomponents</groupId>
            <artifactId>fluent-hc</artifactId>
            <version>4.5.2</version>
        </dependency>

        <dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-test</artifactId>
            <scope>test</scope>
        </dependency>
        <dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-web</artifactId>
        </dependency>
    </dependencies>
```

![image](https://user-images.githubusercontent.com/33775732/236589702-f23f6744-6554-489b-900a-575a9d2aec4c.png)

![image](https://user-images.githubusercontent.com/33775732/236589694-1baa5809-1c68-4aaa-9a41-825f74c00a08.png)

**Comment by YongwuHe** [comment]
you need to use the following method to reproduce:
`org.apache.http.client.fluent.Request.Post(uri).bodyString("xx", ContentType.APPLICATION_JSON).execute()`.

I have fixed this problem, you can refer to it.
https://github.com/arextest/arex-agent-java/pull/155/files

**Comment by YongwuHe** [comment]
Fixed in PR: #155 , this issue will be closed. If you have any questions, please reopen it.

**Comment by YongwuHe** [comment]
Fixed in PR: #155 , this issue will be closed. If you have any questions, please reopen it.
